tests_integration_with_local_chain/markets/seer/test_seer_agent_market.py

```
@@ -58,7 +58,7 @@ def test_seer_place_bet(
     agent_market = check_not_none(agent_market)
     amount = USD(10.0)
 
-    with pytest.raises(Exception) as e:
+    with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)